etna/lib/helpers.rb

```diff
@@ -3,33 +3,57 @@
 
 module WithEtnaClients
   def environment
-    EtnaApp.instance.environment
+    Etna::EtnaApp.instance.environment
   end
 
   def token
-    if !ENV['TOKEN']
-      puts "No environment variable TOKEN is set. Set your token with `export TOKEN=<your.janus.token>` before running Etna commands."
+    env_token = ENV['TOKEN']
+    if !env_token
+      puts "No environment variable TOKEN is set.  You should set your token with `export TOKEN=<your.janus.token>` before running."
+      redirect = EtnaApp.instance.config(:auth_redirect)
+
+      if redirect.nil? && EtnaApp.instance.environment == :production
+        redirect = 'https://janus.ucsf.edu/'
+      end
+
+      unless redirect.nil?
+        puts "Open your browser to #{redirect} to complete login and copy your token."
+      end
+
       exit
     end
-    ENV['TOKEN']
+
+    env_token
   end
 
   def magma_client
-    @magma_client ||= Etna::Clients::Magma.new(token: token, host: EtnaApp.instance.config(:magma, environment)[:host])
+    @magma_client ||= Etna::Clients::Magma.new(
+        token: token,
```

Comment:
I think all of these clients need to pick up the `ignore_ssl` flag from the config? So a line like:

```
        ignore_ssl: EtnaApp.instance.config(:ignore_ssl, environment),
```

In addition to the token and host config seems to be required...

Reply:
A similar change needs to happen for the re-constructed Magma client in the `create_project_from_json_workflow.rb`, [here](https://github.com/mountetna/monoetna/pull/118/files#diff-bb3d2b56b929a0bebb5476ab6f45cda486285d933dc058d831ef9816f606790bR75). So we'll have to make the `ignore_ssl` setting visible on `magma_client` and pass that along.

Reply:
Right, I have a plan for this.  I add the ignore_ssl into the environment config, now I just need to pull it in here.

Reply:
https://github.com/mountetna/monoetna/pull/118/files#diff-56d68a588eadb83c05fceb36e05586003748dd14805266bdab533c767b59746eR55

https://github.com/mountetna/monoetna/pull/118/files#diff-d0be4b3b7a734970e675b48dc7ae203413a1b112a341d1887c9af0423b0368b2R25

https://github.com/mountetna/monoetna/pull/118/files#diff-8d69e0ee4a80de38a29bc3c40276dc35aa103b1020beaed17ee906168fa12777R32

Reply:
:+1: